RELEASE_STEPS.md

```diff
@@ -50,7 +50,11 @@ Prepare changes for the release:
 - Once all changes are done, build locally (and/or verify changes on CI), including:
   - Verify if scanners/linters/checkers passed
   - Verify if version is set properly, especially in `.dll` and `.so` files
+- Create/update a VERSION file for GitHub ZIP downloads (users without git):
+  - `echo "$VERSION" > VERSION`
```

Comment:
hmm actually, wondering - should it be rather `$VERSION+1`...? This would align with our `vX.X.X-devX` approach (where main is introduced as next release)